self_teach_challenge_1.1/script.js

```diff
@@ -12,7 +16,53 @@ function stopCountdown() {
     }
 };
 
+function countdownSettings() {
+    if (disable) {
+        document.getElementById('minutes').disabled = false;
+        document.getElementById('seconds').disabled = false;
+        disable = false;
+    } else {
+        document.getElementById('minutes').disabled = true;
+        document.getElementById('seconds').disabled = true;
+        disable = true;
+    }
+}
+
+function changeButtonValue() {
+    let button = document.getElementById('timer-toggle-button').value;
+    if (clicked) {
+        /*button.value ='STOP' */
+        document.getElementById('timer-toggle-button').value = 'STOP';
+        clicked = false;
+    } else {
+        /*button.value ='START' */
+        document.getElementById('timer-toggle-button').value = 'START';
+        clicked = true;
+    }
+    document.getElementById('minutes').disabled = true;
+    document.getElementById('seconds').disabled = true;
+}
+
+function oneNumber() {
+    if (minutesInput.value.length > 2 || secondsInput.value.lenght > 2) {
+        alert('Helyesen add meg az időt mm:ss')
+        clearInterval(interval)
+    }
+    if (minutesInput.value.length < 2) {
+        minutesInput.value = "0" + minutesInput.value;
+    };
+    if (secondsInput.value.length < 2) {
+        secondsInput.value = "0" + secondsInput.value;
+    };
+    if (countdownSeconds > 5999) {
```

Comment:
Do: Refactor this magic number into a variable. `const purposeOfThisNumber = 5999`